src/nft/components/profile/view/FilterSidebar.tsx

```diff
@@ -99,16 +120,18 @@ const CollectionSelect = ({
             collectionSearchText={collectionSearchText}
             setCollectionSearchText={setCollectionSearchText}
           />
-          <Box paddingBottom="8" overflowY="scroll" style={{ scrollbarWidth: 'none' }}>
-            {displayCollections?.map((collection, index) => (
-              <CollectionItem
-                key={index}
-                collection={collection}
-                collectionFilters={collectionFilters}
-                setCollectionFilters={setCollectionFilters}
-              />
-            ))}
-          </Box>
+          <ItemsContainer>
+            <Box paddingBottom="8" style={{ scrollbarWidth: 'none' }}>
```

Comment:
this overflowY made a random scrollbar show up when it wasn't needed and when there was no overflow